src/apps/vpn/ui/screens/settings/ViewDNSSettings.qml

```diff
@@ -35,6 +35,25 @@ VPNViewBase {
         Layout.rightMargin: VPNTheme.theme.windowMargin
         Layout.topMargin: VPNTheme.theme.windowMargin / 2
 
+        InformationCard {
```

Comment:
Can we stick this in a loader (active on same conditions as visible, and remove visible)

Reply:
yep. better!